### ISSUE TITLE
Tonel loading issue with ZIP archive: can't find EOCD position

### ISSUE THREAD

**Issue by JurajKubelka** [JurajKubelka/DiscordSt#17]
By executing this ([comes from here](https://pharo-link.herokuapp.com/v2/?data=H4sIAAAAAAAAAK3N2wlCMQwA0P8LTiI26dPENZygtIVb8LbB5svpRV3BBc7ROdrjvKvKugGUOkztq8xnzSKmzAOyai770YYu8IjoEDlSCEhMMUGwNgaOyV%2BZKXnHCb6keXU5bWXWdh9dpOk/Dks%2BeJvYwQe%2BrJ9slr4B90hWO8YAAAA%3D), [Tonel ZIP file](https://cdn.discordapp.com/attachments/300020095844089856/411549563799863296/tonel.zip)):

```Smalltalk
 #DSLoadIt asClassIfAbsent: [
    Metacello new
       baseline: #DiscordSt;
       repository: 'github://JurajKubelka/DiscordSt:dev/src';
       load ].
 #DSLoadIt asClassIfPresent: [ :theClass | theClass data: 'H4sIAAAAAAAAAK3N2wlCMQwA0P8LTiI26dPENZygtIVb8LbB5svpRV3BBc7ROdrjvKvKugGUOkztq8xnzSKmzAOyai770YYu8IjoEDlSCEhMMUGwNgaOyV+ZKXnHCb6keXU5bWXWdh9dpOk/Dks+eJvYwQe+rJ9slr4B90hWO8YAAAA=' ].
```

It produces this error:

```
ZipArchive(Object)>>error:
ZipArchive>>readFrom:
DSOpenTonelArchiveCommand>>execute
[ otherWindows
	add:
		(DSOpenTonelArchiveCommand new
			bytes: aDSSharedTonel bytes;
			execute;
			window) ] in DSDisplaySharedPlayground>>visitDSSharedTonel: in Block: [ otherWindows...
ByteSymbol(Symbol)>>value:
OrderedCollection>>do:
DSDisplaySharedPlayground>>execute
DSLoadIt>>displayToUser:
DSLoadIt>>execute
DSLoadIt class>>data:
[ :theClass | 
theClass
	data:
		'H4sIAAAAAAAAAK3N2wlCMQwA0P8LTiI26dPENZygtIVb8LbB5svpRV3BBc7ROdrjvKvKugGUOkztq8xnzSKmzAOyai770YYu8IjoEDlSCEhMMUGwNgaOyV+ZKXnHCb6keXU5bWXWdh9dpOk/Dks+eJvYwQe+rJ9slr4B90hWO8YAAAA=' ] in UndefinedObject>>DoIt in Block: [ :theClass | ...
BlockClosure>>cull:
ByteSymbol(String)>>asClassIfPresent:
UndefinedObject>>DoIt
OpalCompiler>>evaluate
RubSmalltalkEditor>>evaluate:andDo:
RubSmalltalkEditor>>highlightEvaluateAndDo:
[ textMorph textArea editor highlightEvaluateAndDo: ann action.
textMorph shoutStyler style: textMorph text ] in [ textMorph textArea
	handleEdit: [ textMorph textArea editor highlightEvaluateAndDo: ann action.
		textMorph shoutStyler style: textMorph text ] ] in GLMMorphicPharoScriptRenderer(GLMMorphicPharoCodeRenderer)>>actOnHighlightAndEvaluate: in Block: [ textMorph textArea editor highlightEvaluateAndDo...etc...
RubEditingArea(RubAbstractTextArea)>>handleEdit:
[ textMorph textArea
	handleEdit: [ textMorph textArea editor highlightEvaluateAndDo: ann action.
		textMorph shoutStyler style: textMorph text ] ] in GLMMorphicPharoScriptRenderer(GLMMorphicPharoCodeRenderer)>>actOnHighlightAndEvaluate: in Block: [ textMorph textArea...
WorldState>>runStepMethodsIn:
WorldMorph>>runStepMethods
WorldState>>doOneCycleNowFor:
WorldState>>doOneCycleFor:
WorldMorph>>doOneCycle
WorldMorph class>>doOneCycle
[ [ WorldMorph doOneCycle.
Processor yield.
false ] whileFalse: [  ] ] in MorphicUIManager>>spawnNewProcess in Block: [ [ WorldMorph doOneCycle....
[ self value.
Processor terminateActive ] in BlockClosure>>newProcess in Block: [ self value....
```

**Comment by JurajKubelka** [comment]
There might be a relation with the failing `testZipFileSubdirs` test case on Linux #15. But he sent it after the fix, so it should not.

**Comment by JurajKubelka** [comment]
I do not know how to fix this as I cannot reproduce the corrupted ZIP creation. Hopefully it was fixed in #15.
